mozilla/blurts-server
# Fix dashboard top banner CTAs

# References: 
Jira:
- [MNTOR-2197](https://mozilla-hub.atlassian.net/browse/MNTOR-2197)
- [MNTOR-2278](https://mozilla-hub.atlassian.net/browse/MNTOR-2278)



# Description

Fixes the following dashboard top banner CTAs:
- Get continuous protection: This should show the Premium upsell dialog.
- Check more email addresses: Should link to `/redesign/user/settings`.

Not applicable.

# How to test

- Get continuous protection CTA: http://localhost:6006/?path=/story/pages-dashboard--dashboard-us-no-premium-empty-scan-no-breaches
- Check more email addresses CTA: http://localhost:6006/?path=/story/pages-dashboard--dashboard-us-no-premium-scan-in-progress-resolved-breaches


[MNTOR-2197]: https://mozilla-hub.atlassian.net/browse/MNTOR-2197?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MNTOR-2278]: https://mozilla-hub.atlassian.net/browse/MNTOR-2278?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
